### ISSUE TITLE
Run the android emulator an error（only Android）

### ISSUE THREAD

**Issue by houqinzhe** [Sh1d0w/multi_image_picker#165]
**Bug Report**

`multi_image_picker` version: 4.3.3

Is it happening on Android:  Yes

Is it happening on iOS:  No

```
[✓] Flutter (Channel dev, v1.6.5, on Mac OS X 10.14.5 18F132, locale zh-Hans-CN)

[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 10.2.1)
[✓] iOS tools - develop for iOS devices
[✓] Android Studio (version 3.4)
[!] IntelliJ IDEA Ultimate Edition (version 2018.3.3)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] Connected device (1 available)
```

**Describe the bug**

java.lang.ClassNotFoundException: Didn't find class "com.sangcomz.fishbun.FishBunFileProvider"

**Comment by Sh1d0w** [comment]
Can't reproduce, clean your flutter cache and re install the plugin.

**Comment by LiewJunTung** [comment]
It happens to me as well. Here is how I produce the error.
1. Android Studio -> Create A Flutter Project -> Update pubspec.yaml with `multi_image_picker: ^4.3.3`
2. Run it an Android phone.

Tried cleaning the cache and reinstalling the plugin. 

Error: 
java.lang.RuntimeException: Unable to get provider com.sangcomz.fishbun.FishBunFileProvider: java.lang.ClassNotFoundException: Didn't find class "com.sangcomz.fishbun.FishBunFileProvider"
<img width="1355" alt="Screenshot 2019-06-05 at 11 26 52 AM" src="https://user-images.githubusercontent.com/4220976/58928395-e9d51700-8784-11e9-95a4-1bde32198214.png">

**Comment by Sh1d0w** [comment]
@LiewJunTung Thanks, I was able to reproduce.

The problem is because the Android plugin I am using haven't migrated to Android X. I will open a PR.

Meanwhile we are waiting for it to be released, you can fix this by opening your flutter android folder with Android Studio then from the menu click "Refactor" -> Migrate to Android X.

Then your project will run.

**Comment by Sh1d0w** [comment]
Here is a reference to the PR that fixes the issue https://github.com/sangcomz/FishBun/pull/158

**Comment by LiewJunTung** [comment]
Ah, I tried upgrading to AndroidX. Yes, that solved this problem. Thanks @Sh1d0w! :)

**Comment by kdela** [comment]
@Sh1d0w I see this bug is already resolved from FishBun side. Do you have any idea when it will be released fixed multi_image_picker? It blocks any android release

**Comment by Sh1d0w** [comment]
@kdela Yes it was resolved and merged, but it was not released. Latest version of FishBun is 0.9.1, which I am currently using. Once the maintainer releases new version I will update it as well.

This should not block your release, follow the steps I've suggested in this issue to manually migrate to AndroidX and it should work.

**Comment by pratik037** [comment]
> @kdela Yes it was resolved and merged, but it was not released. Latest version of FishBun is 0.9.1, which I am currently using. Once the maintainer releases new version I will update it as well.
> 
> This should not block your release, follow the steps I've suggested in this issue to manually migrate to AndroidX and it should work.

Is the new version released now?

**Comment by Sh1d0w** [comment]
@pratik037 Yes it should be fixed in the latest version 4.5.5
